### ISSUE TITLE
No options to backup configurations to file

### ISSUE THREAD

**Issue by fpiraneo** [termius/termius-cli#206]
I need to migrate from one mac to another; there are no options to backup all the setup from one machine to another. Pls provide exhaustive documentation to backup the data and restore on another host.

**Comment by viplo** [comment]
Yep unfortunately, without subscription your are not able to do that.
And for me it is a no way for this superb app. I used it since three years then my computer crash, I had to change it so I moved to tabby
Bye bye Termius
